Sources/MapboxDirectionsCLI/ProcessCommand.swift

```diff
@@ -53,6 +56,29 @@ class ProcessCommand<ResponceType : Codable, OptionsType : DirectionsOptions > :
                let jsonData = try? JSONSerialization.data(withJSONObject: object, options: [.prettyPrinted]) {
                 outputText = String(data: jsonData, encoding: .utf8)!
             }
+        case .gpx:
+            var gpxText: String = String("<?xml version=\"1.0\" encoding=\"UTF-8\"?>")
```

Comment:
Maybe it would be better to use an array of strings and to join them at the end of the case? I heard that it works faster and you don't have to start each line with `\n` manually.

Reply:
Also, if you find the backslashes annoying, you can replace each `"` with a `'`, because single-quote-delimited attributes are also valid XML.

(A more ambitious refactor could use the object-oriented [XMLDocument](https://developer.apple.com/documentation/foundation/xmldocument) API or a Swiftier library like [XMLCoder](https://github.com/MaxDesiatov/XMLCoder/), but an extra dependency might be overkill for this simple XML document.)